pallets/staking/src/lib.rs

```diff
@@ -2081,40 +2114,39 @@ decl_module! {
             ValidatorCount::mutate(|n| *n += factor * *n);
         }
 
-        /// Governance committee on 2/3 rds majority can introduce a new potential validator
-        /// to the pool of validators. Staking module uses `PermissionedValidators` to ensure
-        /// validators have completed KYB compliance and considers them for validation.
+        /// Governance committee on 2/3 rds majority can introduce a new potential entity
+        /// to the pool of complianced entities who can run validators. Staking module uses `PermissionedEntities`
+        /// to ensure validators have completed KYB compliance and considers them for validation.
```

Comment:
"KYB" -> "KYC"?

Reply:
It is fine KYB stands for KnowYourBusiness